lib/src/multires_stochastic_texture_synthesis.rs

```diff
@@ -817,6 +813,10 @@ impl Generator {
                         let max_candidate_count = params.nearest_neighbors as usize
                             + params.random_sample_locations as usize;
 
+                        let my_thread_num = thread_counter.fetch_add(1, Ordering::Relaxed);
```

Comment:
Nitpick: should be `my_thread_id` for consistency with other ordinals.

Reply:
👍